src/components/UserInputBlock.tsx

```diff
@@ -51,12 +66,45 @@ export const UserInputBlock = ({ content, prompt, ack, waitForEvent, fireEvent,
       .max(MAX_CHARACTER_INPUT_BLOCK_LENGTH, EXCEEDED_MAX_INPUT_ERROR)
   })
 
+  const getPersistedState = async (): Promise<void> => {
+    try {
+      if (contentId === undefined || persistor === undefined) {
+        setPersistorGetStatus(PersistorGetStatus.Success)
+        return
+      }
+
+      const persistedState = await persistor.get(contentId)
+      if (persistedState !== null) {
+        const parsedLocalStorage = JSON.parse(persistedState)
```

Comment:
Ah, missed this minor name nit to remove references to local. Maybe `parsedPersistedState`?

Reply:
I told myself to update that and forgot to address after our huddle so no worries. Thanks for catching. Updated in latest commit.